visualization/app/codeCharta/state/nodeSearch.service.spec.ts

```diff
@@ -46,11 +45,6 @@ describe("NodeSearchService", () => {
 			nodeSearchService["codeMapPreRenderService"].getRenderMap = jest.fn(() => {
 				return JSON.parse(JSON.stringify(TEST_FILE_WITH_PATHS.map))
 			})
-
-			// Workaround for Windows paths, since "ignore" does not work for unit tests
```

Comment:
We should fix that if it does not work on Windows. Otherwise this just masks an actual problem.